app/components/ui/GDocEmbed/GDocEmbed.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Iframe from 'react-iframe';
+import Footer from '../Footer/Footer';
+
+import styles from './GDocEmbed.scss';
+
+
+export default class GDocEmbed extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+    };
+  }
+
+  render() {
+    const { pageHeader } = this.props;
+    const { GDocURL } = this.props;
+    return (
+      <React.Fragment>
+        <article className={styles.GPage}>
+          <header className={styles.GHeader}>
+            <h1>
+              {pageHeader}
+            </h1>
+            <p>
+              This list is compiled to link individuals experiencing homelessness to resources.
+              We will be updating the list continuously, when services change.
+              <br />
+              Please email
+              <a href="mailto:contact@sheltertech.org">
+                {' '}
+                contact@sheltertech.org
+              </a>
+              {' '}
+              to suggest edits.
+            </p>
+          </header>
+          <section>
+            <Iframe
+              title={pageHeader}
+              url={GDocURL}
+              width="80%"
+              height="100%"
+              padding="0pt"
+              allowFullScreen="yes"
+              className={styles.GDocEmbed}
+            />
+          </section>
+        </article>
+        <Footer />
```

Comment:
I don't feel that this footer belongs here. The component should only be responsible for embedding a single Google Document. It should be the responsibility of the page component that includes this GDocEmbed to insert a footer.

One example of a problem with this approach is that if you wanted to include two GDocEmbeds into the same page, you'd end up getting two footers. Or, if you wanted to include a GDocEmbed on a different page, but you wanted a different, perhaps shorter footer, you'd be stuck using the same footer style on both GDocEmbeds.